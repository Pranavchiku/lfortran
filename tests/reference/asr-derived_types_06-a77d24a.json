{
    "basename": "asr-derived_types_06-a77d24a",
    "cmd": "lfortran --indent --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/derived_types_06.f90",
    "infile_hash": "755cad69ac7bfd9d2d069a753b811e2f76bec02faf78f74031b9f2c1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-derived_types_06-a77d24a.stdout",
    "stdout_hash": "c15dce6d0e99bdd11f1efec3f7373404297be8a6489f5f0dd2f1f3c4",
    "stderr": "asr-derived_types_06-a77d24a.stderr",
    "stderr_hash": "824c09df494b86e34c0f91dee95f5be317400110d5c2df5d7b5466e2",
    "returncode": 0
}