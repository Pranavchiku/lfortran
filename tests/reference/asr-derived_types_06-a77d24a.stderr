warning: Implied save feature
 --> tests/../integration_tests/derived_types_06.f90:7:5
  |
7 |     class(*), intent(out) :: a
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^ Assuming save attribute implicitly for variables
